Source/WebCore/dom/Element.cpp

```diff
@@ -5363,4 +5379,28 @@ bool Element::isPopoverShowing() const
     return popoverData() && popoverData()->visibilityState() == PopoverVisibilityState::Showing;
 }
 
+// https://drafts.csswg.org/css-contain/#relevant-to-the-user
+bool Element::isRelevantToUser() const
+{
+    return hasRareData() && elementRareData()->contentRelevancyStatus();
```

Comment:
Would be more readable with an explicit `!contentRelevancyStatus().isEmpty()`

Reply:
Also why doesn't this just call `contentRelevancyStatus()`?

Reply:
Right, that is much simpler, done.